src/components/Grid/Row.module.css

```diff
@@ -42,22 +42,14 @@
 
 /* Rows below header */
 .row:not(:first-child):hover {
-  /* TODO -- make this Ethos specific */
-  --default-row-hover-background: #deebff;
+  --default-row-hover-background: var(--BrandDuckegg);
 
   background-color: var(
     --ariagrid-row-hover-background,
     --default-row-hover-background
   );
 }
 
-.row:not(:first-child):hover > div:first-child {
```

Comment:
I was looking at Zendesk's table when I did this since Tom likes those. This CSS put a left border on the row when hovered, but was "too much like" their tables and so I removed it :)